website/views.py

```diff
@@ -252,46 +265,48 @@ def workshop_create(request, event_id, slug):
     Returns:
         HTTP response containing the workshop creation page
     """
-    if request.method == 'POST':
-        workshop_form = WorkshopForm(request.POST, prefix='workshop_form')
-        if workshop_form.is_valid():
-            workshop_form.save()
-            return redirect('website:event_page', slug=slug, event_id=event_id)
-    else:
-        workshop_form = WorkshopForm(prefix='workshop_form')
-        workshop_form['event'].initial = get_object_or_404(Event, pk=event_id)
+    form_class = WorkshopForm
+    template_name = 'website/workshop_create.html'
 
-    context = {
-        'workshop_form': workshop_form,
-        'event': get_object_or_404(Event, pk=event_id)
-    }
-    return render(request, 'website/workshop_create.html', context)
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
 
+        event = Event.objects.get(id=self.kwargs['event_id'])
+        form = self.get_form()
+        form.fields['event'].initial = event
 
-def about(request):
-    """
-    Render and show the about page.
+        context['workshop_form'] = form
+        context['event'] = event
 
-    Args:
-        request: HTTP request header contents
-
-    Returns:
-        HTTP response containing the about page
-    """
-    return render(request, 'website/about.html')
+        return context
 
+    def get_success_url(self):
+        return reverse('website:event_page', kwargs=self.kwargs)
 
-@login_required
-def user_profile(request):
+class About(TemplateView):
```

Comment:
Maybe rename this to `AboutView`?
```suggestion
class AboutView(TemplateView):
```
Remember to change the view reference in `urls.py` as well.